_includes/about.html

```diff
@@ -1,10 +1,10 @@
 <p>CivicActions Accessibility is an open project primarily developed by CivicActions team members.</p>
 
-<p>We follow [agile process](/agile) and encourage others to do so as well.</p>
+<p>We follow <a href="https://accessibility.civicactions.com/agile">agile process</a> and encourage others to do so as well.</p>
 
 <p>We are gathering information that we find useful for our team, and hope that it resonates with our clients and the broader community. We acknolwedge that this resource is imperfect, impermanent and incomplete, and we encourage you to help make it better. There is a lot of great information about digital accessibility already freely available on the web. We aim to refer people to great resources, or build upon them.</p>
 
-<p>Please [let us know](https://accessibility.civicactions.com/contact) what you find helpful, if there are resources we should include or if there are ways we can improve.</p>
+<p>Please <a href="https://accessibility.civicactions.com/contact">let us know</a>> what you find helpful, if there are resources we should include or if there are ways we can improve.</p>
```

Comment:
Looks like an extra `>` was left behind. Last change and this is good to go!

Reply:
Opps. Nice catch!!